src/Dflydev/FigCookies/SetCookies.php

```diff
@@ -70,13 +71,12 @@ public function without(string $name) : SetCookies
 
     /**
      * Render SetCookies into a Response.
-     *
      */
-    public function renderIntoSetCookieHeader(ResponseInterface $response) : ResponseInterface
+    public function renderIntoSetCookieHeader(ResponseInterface $response): ResponseInterface
     {
-        $response = $response->withoutHeader(static::SET_COOKIE_HEADER);
+        $response = $response->withoutHeader(self::SET_COOKIE_HEADER);
         foreach ($this->setCookies as $setCookie) {
-            $response = $response->withAddedHeader(static::SET_COOKIE_HEADER, (string) $setCookie);
+            $response = $response->withAddedHeader(self::SET_COOKIE_HEADER, (string) $setCookie);
```

Comment:
changes are low (http spec), but also this constant could be overridden

Reply:
I think I've completely underestimated the `static::` vs `self::` implications in a few places... Is doing `static::` bad here for some reason?